seedfarmer/commands/_module_commands.py

```diff
@@ -105,6 +105,13 @@ def _prebuilt_bundle_check(mdo: ModuleDeployObject) -> Optional[str]:
         return None
 
 
+def _get_codebuild_fleet_arn(deployment_manifest: DeploymentManifest, region: str, account_id: str) -> Optional[str]:
+    for tam in deployment_manifest.target_account_mappings:
+        if tam.codebuild_fleet:
```

Comment:
I think you need to alter the condition here in case more than one TAM has an associated CodeBuild fleet?
```
if tam.codebuild_fleet and tam.account_id == account_id and tam.region == region
```